### PR TITLE
Add zkSend SDK Docs

### PR DESCRIPTION
**Pull request by Jordan-Mysten**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and do not break anything, you can skip the following section. Otherwise, please briefly describe what has changed under the Release Notes section.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: sdk/docs/pages/zksend/dapp.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+To use the zkSend wallet, you will need to register it in your application, using
+`registerZkSendWallet`. This only needs to be done once, and should be done as early as possible in
+your application's lifecycle.
+
+`registerZkSendWallet` takes two arguments:
+
+- `name`: The name of your dApp. This will be shown to the user when they are asked to approve the
+  connection in zkSend.
+- `options`: An optional object with the following properties:
+  - `origin`: The origin of the zkSend website. Defaults to `https://zksend.com`.
```
Comment (plam-ml) [inline sdk/docs/pages/zksend/dapp.mdx:28]: extra spacing intended?

File: sdk/docs/pages/zksend/link-builder.mdx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+</Callout>
+
+You can start creating your own zkSend link using the `ZkSendLinkBuilder` class. This class
+constructor takes an object with the following options:
+
+- **`sender`** (required) - Required. The address of the sender / creator of the link.
+- **`redirect`** (optional) - The redirect configuration for the link. See "Claim redirect" below
+  for more information.
+  - **`url`** - The url to redirect to after the link have been claimed.
+  - **`name`** - The name of your dApp. This will be shown to the user when claiming the link.
```
Comment (plam-ml) [inline sdk/docs/pages/zksend/link-builder.mdx:34]: same here with spacing, wonder if intended

File: sdk/docs/pages/zksend/dapp.mdx (hunk 1 above)
Comment (Jordan-Mysten) [reply sdk/docs/pages/zksend/dapp.mdx:28]: Yeah it's nested under `options`

File: sdk/docs/pages/zksend/dapp.mdx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+Using the zkSend SDK, you can allow users to connect to the zkSend wallet from your dApp. The wallet
+is provided through the [Wallet Standard](https://docs.sui.io/standards/wallet-standard), and should
+appear automatically in your existing wallet connection UI.
+
+## Considerations
+
+- zkSend only supports mainnet at this time.
+- Users will only be able to connect their zkLogin wallet from zkSend.
+- The zkLogin account will be managed by zkSend, you do not need to set up any oAuth providers.
```
Comment (williamrobertson13) [inline sdk/docs/pages/zksend/dapp.mdx]:
```suggestion
- The zkLogin account will be managed by zkSend, you do not need to set up any OAuth providers.
```

File: sdk/docs/pages/zksend/dapp.mdx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+    ```ts
+    registerZkSendWallet('Your dApp Name', { origin: 'https://preview.zksend.com' });
+    ```
+
+</Callout>
+
+## Supported features
+
+The zkSend wallet currently supports the following features
```
Comment (williamrobertson13) [inline sdk/docs/pages/zksend/dapp.mdx]:
```suggestion
The zkSend wallet currently supports the following features:
```

File: sdk/docs/pages/zksend/dapp.mdx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+- `signTransactionBlock`
+- `signPersonalMessage`
+
+We intentionally do not support `signAndExecuteTransactionBlock`, and suggest that your dApp instead
+use `signTransactionBlock`, and submit the signed transaction within your dApp instead. This
+provides better control around execution, and avoids potential issues around data inconsistency. If
```
Comment (williamrobertson13) [inline sdk/docs/pages/zksend/dapp.mdx:54]: a very minimal code example might be nice

**Review by williamrobertson13 — APPROVED** [review]
skimmed through and looked pretty good to me! nice and thorough

**Pull request merged**